tools/Google.Cloud.Tools.VersionCompat/CecilUtils/ShowExtensions.cs

```diff
@@ -43,13 +55,26 @@ private static string Show(this ParameterDefinition p)
             if (p.IsOptional)
             {
                 sb.Append(" = ");
-                sb.Append(p.Constant ?? "null"); // TODO: I suspect `Constant` is not thw right way to do this.
+                sb.Append(p.Constant ?? "null"); // TODO: I suspect `Constant` is not the right way to do this.
             }
             return sb.ToString();
         }
 
-        public static string Show(this TypeReference type) =>
-            type.FullName == "System.Void" ? "void" : $"{type.FullName.Split('`')[0].Replace('/', '.')}{type.GenericParameters.Show()}";
+        public static string Show(this TypeReference type)
```

Comment:
As a nit, I think Format would be a better name than Show. But that's not really a matter of this PR.

Reply:
ack